src/main/kotlin/org/rust/ide/hints/RsInlayTypeHintsProvider.kt

```diff
@@ -126,6 +130,20 @@ class RsInlayTypeHintsProvider : InlayHintsProvider<RsInlayTypeHintsProvider.Set
                 }
             }
 
+            private fun presentTypeArguments(declaration: RsLetDecl) {
```

Comment:
Maybe `presentTypePlaceholders`?

Reply:
Also, please add a setting enabled by default for this hint kind (see `org.rust.ide.hints.RsInlayTypeHintsProvider.Settings`). It allows the user to disable some kinds of hints.